agent/containers/images/visualizers/live-metric-visualizer/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM fedora:latest
```

Comment:
This looks quite similar to the `prom-graf-visualizer` container.  Any way we can reduce the overlap?  For instance, perhaps have `prom-graf-visualizer` build its image `FROM live-metric-visualizer`?